docs/src/main/sphinx/connector/mongodb.md

```diff
@@ -210,9 +210,9 @@ This property is optional; the default is `0`.
 MongoDB maintains table definitions on the special collection where `mongodb.schema-collection` configuration value specifies.
 
 :::{note}
-There's no way for the plugin to detect a collection is deleted.
-You need to delete the entry by `db.getCollection("_schema").remove( { table: deleted_table_name })` in the Mongo Shell.
-Or drop a collection by running `DROP TABLE table_name` using Trino.
+The plugin cannot detect that a collection has been deleted. You must delete the
+entry in the MongoDB Shell using
+`db.getCollection("_schema").remove( { table: deleted_table_name })`, or drop a collection in Trino by running `DROP TABLE table_name`.
```

Comment:
nit: what do you think about: "by executing <command> in the MongoDB Shell"?

Reply:
I like that.